.github/workflows/lint.yml

```diff
@@ -15,19 +15,17 @@ jobs:
       - name: Check out Git repository
         uses: actions/checkout@v2
 
-      - name: Set up node
-        uses: actions/setup-node@v1
-        with:
-          node-version: 16.x
-          registry-url: https://registry.npmjs.org
+      - name: Set the correct Node version using nvm
+        shell: bash -l {0} 
```

Comment:
What does this mean? Why not just set `bash`?

Reply:
I didn't spend much thought into this one tbh. I looked at this [article](https://snippets.khromov.se/use-nvm-in-github-actions-when-building-a-node-js-project/), and decided to use it verbatim. For the reproducibility, I didn't opt into the github action nvm from the [marketplace](https://github.com/marketplace/actions/set-up-node-using-nvm).

Will try using `bash` only and see if it works.

Reply:
It appears only setting bash will fail, see failed workflow [run](https://github.com/Uniswap/routing-api/actions/runs/4995256903/jobs/8946899274?pr=190) vs successful workflow [run](https://github.com/Uniswap/routing-api/actions/runs/4994295293/jobs/8944653089). 

I noticed in the failed run:
```
nvm install
  shell: /usr/bin/bash --noprofile --norc -e -o pipefail {0}
/home/runner/work/_temp/9e1ec01[4](https://github.com/Uniswap/routing-api/actions/runs/4995256903/jobs/8946899274?pr=190#step:3:5)-ee21-422d-a00b-8f226d4a076d.sh: line 1: nvm: command not found
Error: Process completed with exit code 127.
```

in the successful run:
```
nvm install
  shell: /usr/bin/bash -l {0}
Found '/home/runner/work/routing-api/routing-api/.nvmrc' with version <18.1[3](https://github.com/Uniswap/routing-api/actions/runs/4994295293/jobs/8944653089#step:3:3).0>
Downloading and installing node v18.13.0...

```